db/descriptions/schema.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2023 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package descriptions
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/ipfs/go-datastore/query"
+
+	"github.com/sourcenetwork/defradb/client"
+	"github.com/sourcenetwork/defradb/core"
+	"github.com/sourcenetwork/defradb/core/cid"
+	"github.com/sourcenetwork/defradb/datastore"
+)
+
+// CreateSchemaVersion creates and saves to the store a new schema version.
```

Comment:
suggestion: "a new schema description version"

Reply:
I do disagree, and don't think adding `description` there adds any clarity.  It does also clash with Fred's long-term naming suggestion where `SchemaDescription` will be renamed to `Schema` (or maybe `SchemaVersion`, either way `Description` will be dropped).

Reply:
it was just observation as a reader that I was looking for "description" word because the method is in `descriptions` package, but it wasn't clear to me at first sight why.